frontend/packages/console-app/src/components/cloud-shell/CloudShellDrawer.tsx

```diff
@@ -41,6 +47,9 @@ const CloudShellDrawer: React.FC<CloudShellDrawerProps> = ({ children, onClose }
             href="/terminal"
             target="_blank"
             aria-label={t('console-app~Open terminal in new tab')}
+            onClick={() => {
+              fireTelemetryEvent('Terminal Expanded');
+            }}
```

Comment:
Do we care that the user clicked the button? This can be captured via URL.

BTW to be consistent with other events, it should start with `Web Terminal`. That is if we are keeping this event.

Reply:
if I got it right,  we don't need this to be captured as an event as eventType page already captures it. https://github.com/openshift/console/blob/master/frontend/public/components/app.jsx#L288